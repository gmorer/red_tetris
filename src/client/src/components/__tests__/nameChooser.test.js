import NameChooser from './../nameChooser';
import React from 'react';
import renderer from 'react-test-renderer';

test('Some test', () => {
	const component = renderer.create(<NameChooser />)
	const tree = component.toJSON();
	expect(tree).toMatchSnapshot();
})