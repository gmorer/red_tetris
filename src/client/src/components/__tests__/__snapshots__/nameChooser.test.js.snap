// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Some test 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "backgroundImage": "url('bg2.jpg')",
      "backgroundRepeat": "no-repeat",
      "backgroundSize": "100% 100%",
      "display": "flex",
      "height": "100%",
      "justifyContent": "center",
      "textAlign": "center",
      "width": "100%",
    }
  }
>
  <form
    onSubmit={[Function]}
  >
    <input
      autoFocus={true}
      id="name_input"
      placeholder="Enter your Name"
      style={
        Object {
          "border": "2px solid black",
          "borderRadius": "25px",
          "boxSizing": "border-box",
          "padding": "12px 20px",
          "width": "100%",
        }
      }
    />
    <button
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#9f233a",
          "border": "2px solid white",
          "borderRadius": "8px",
          "color": "white",
          "cursor": "pointer",
          "height": "3em",
          "marginTop": "1em",
          "textDecoration": "none",
          "width": "15em",
        }
      }
    >
      OK
    </button>
  </form>
</div>
`;
